accounts-db/src/accounts_db.rs

```diff
@@ -394,10 +394,10 @@ impl CurrentAncientAccountsFile {
         let accounts = accounts_to_store.get(storage_selector);
 
         let previous_available = self.accounts_file().accounts.remaining_bytes();
-        let timing = db.store_accounts_frozen(
-            (self.slot(), accounts, accounts_to_store.slot()),
-            self.accounts_file(),
-        );
+
+        let accounts = [(accounts_to_store.slot(), accounts)];
+        let storable_accounts = StorableAccountsBySlot::new(self.slot(), &accounts);
```

Comment:
this is ancient appending. possibly 2 different slots involved. `accounts_to_store.slot()` represents the slot `accounts` should be loaded from. This easily fits into the use case for `StorableAccountsBySlot`.